src/client/metadataApiDeploy.ts

```diff
@@ -408,7 +408,11 @@ export class MetadataApiDeploy extends MetadataTransfer<MetadataApiDeployStatus,
       const zip = createArchive('zip', { zlib: { level: 9 } });
       // anywhere not at the root level is fine
       zip.directory(this.options.mdapiPath, 'zip');
-      await zip.finalize();
+      // archiver/zip.finalize looks like it is async, because it extends streams, but it is not meant to be used that way
+      // the typings on it are misleading and unintended.  More info https://github.com/archiverjs/node-archiver/issues/476
+      // If you await it, bad things happen, like the convert process exiting silently.  https://github.com/forcedotcom/cli/issues/1791
+      // leave the void as it is
+      void zip.finalize();
```

Comment:
![image](https://user-images.githubusercontent.com/76090802/201430110-a21ead97-8948-41e9-8803-3f7cff29fdc1.png)

Reply:
It's a shame [gifs](https://media3.giphy.com/media/ycagKBYEmaili/giphy.gif?cid=6104955eom1mk4qazwicjdstjmrxcjdlwykk4dkrnmuethvc&rid=giphy.gif&ct=g) don't work in github comments